### ISSUE TITLE
Question on reproduction

### ISSUE THREAD

**Issue by cshanjiewu** [ictnlp/DSTC8-AVSD#2]
Thank you for sharing your work. I cannot reproduce the results of the paper in practice. May I ask how many epochs were trained during training and how much the final loss converged?

**Comment by lizekang** [comment]
Do you use the 6-reference test set? 4 epochs during training and nll_loss on validation set is about 1.62.

**Comment by cshanjiewu** [comment]
I follow the default setting for training the model.
![Selection_001](https://user-images.githubusercontent.com/42206191/94387368-855d7880-017c-11eb-8cc9-2c8e4f6e1d60.png)
This is the training process. 
![Selection_002](https://user-images.githubusercontent.com/42206191/94387391-973f1b80-017c-11eb-82ae-90acbac678f0.png)
This is the evaluation. The model output the bad response.

**Comment by cshanjiewu** [comment]
I think the model is trained enough by myself, maybe there is something to pay attention to when testing? My beam search is set to 5. I am so confused about the output when testing.

**Comment by lizekang** [comment]
when you are training and testing, what feature do you use?

**Comment by cshanjiewu** [comment]
Just your default setting('vggish', 'i3d_flow', 'i3d_rgb')

**Comment by lizekang** [comment]
how about text features? do you use the caption, summary, and dialogue?

**Comment by cshanjiewu** [comment]
I haven't modified the code yet, just use your original settings, so the text features should be used according to your original code.

**Comment by lizekang** [comment]
Sorry, I check the code and find a bug in VideoGPT2.py. I think I uploaded the wrong version before. Now it will be the right version and you can try again. Hope you can get the right results. If there are still any problems, please don't hesitate to ask me.

**Comment by cshanjiewu** [comment]
Thank you for your patient reply, I will try again.

**Comment by lizekang** [comment]
When testing, you can use the 6-ref test set to compute the metrics.

**Comment by cshanjiewu** [comment]
![image](https://user-images.githubusercontent.com/42206191/94652295-5c7be580-032c-11eb-868a-00b4d528a775.png)
Hello, I modified the vidioGPT.py file as you did, but after the modification, after one epoch during my training, the loss no longer drops. The model cannot converge completely. All outputs are tag when testing.

**Comment by lizekang** [comment]
Sorry about that, I will rerun it at once. I will check it again. Because of something fusion with other projects, the version may be in disorder. I will inform you at once when I find out why.

**Comment by cshanjiewu** [comment]
Ok, Thanks.

**Comment by lizekang** [comment]
![image](https://user-images.githubusercontent.com/15322635/94882277-193f8500-049a-11eb-8a22-514d69ca8cba.png)
I fixed the bug. You can try again. It should be ok. Be sure that using the default settings, including batch_size, learning rate, etc. The nll loss can be about 1.62.

**Comment by cshanjiewu** [comment]
Thanks, I can get a reasonable result.
